run_all_tests.py

```diff
@@ -0,0 +1,3 @@
+import os
+
+os.system('cmd /c "python -m pytest"')
```

Comment:
I'm interested: what does `cmd /c` do?

Reply:
`/c` is basically "exit after the process is done", not sure it would change much here